src/equations_matrices_routines.f90

```diff
@@ -3005,6 +3008,60 @@ END SUBROUTINE EquationsMatrices_JacobianTypesSet
   !================================================================================================================================
   !
 
+  !>Sets/changes the finite difference step size used for calculating the Jacobian
+  SUBROUTINE EquationsMatrices_JacobianFiniteDifferenceStepSizeSet(vectorMatrices,jacobianStepSizes,err,error,*)
+
+    !Argument variables
+    TYPE(EquationsMatricesVectorType), POINTER :: vectorMatrices !<A pointer to the vector equations matrices to set the Jacobian finite difference step sizes for.
+    REAL(DP), INTENT(IN) :: jacobianStepSizes(:) !<jacobianStepSizes(matrixIdx). The Jacobian finite difference step sizes for the matrixIdx'th Jacobian matrix.
```

Comment:
Not sure why multiple step sizes are allowed like this as there is no information on what Jacobian the step size is for? If there were multiple Jacobians and you only specified one step size then what Jacobian are you trying to set the Jacobian for?